sdk/resourcemanager/azure-resourcemanager-perf/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure.resourcemanager</groupId>
+  <artifactId>azure-resourcemanager-perf</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-perf;current} -->
+  <packaging>jar</packaging>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager</artifactId>
+      <version>2.0.0-beta.5</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager;current} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager-test</artifactId>
```

Comment:
Do we need the test framework?

Reply:
AuthFile from azure-resourcemanager-test.
ResourceNamer from azure-core-test.(As per the potential changes may happen on SdkContext.)

We may remove this if we expect to use other credential and SdkContext.

Reply:
`ResourceNamer` is not exactly required. A random GUID will mostly do.

`AuthFile` seems needed unless we choose a different credential.

Reply:
I was putting UUID for creating storage account while the service return errors by resource name restriction.

Reply:
storage account name is globally unique, and cannot have "-" in it. These requirement is there whatever util you choose, and no util can help if the name generated conflict with any one exists...